src/lib.rs

```diff
@@ -47,16 +47,24 @@ macro_rules! register_module {
     ($module:pat, $init:block) => {
```

Comment:
While we are making breaking changes, I suggest changing this to use a more idiomatic lambda syntax pattern. I.e.,

```rust
register_module!(|cx| {
    // ...
})
```